source/fundamentals/auth.txt

```diff
@@ -318,6 +329,31 @@ mechanism:
         automatically picked up by your MongoClient when you specify the
         ``MONGODB-AWS`` authentication mechanism.
 
+Refresh Credentials
++++++++++++++++++++
+
+The driver supports refreshing credentials for cases such as assuming roles
+or using `Elastic Kubernetes Service <https://docs.aws.amazon.com/eks/latest/userguide/what-is-eks.html>`__.
+
+
+.. code-block:: java
+   :emphasize-lines: 3, 7
+
+   Supplier<AwsCredential> awsFreshCredentialSupplier = () -> {
+       // code to fetch fresh credentials, such as assuming a role using the AWS SDK
```

Comment:
Since these two lines aren't a single sentence, let's add some punctuation/capitalization to make in more readable.